spikeinterface/core/curationsorting.py

```diff
@@ -0,0 +1,119 @@
+from .mergesorting import MergeSorting
+from .splitsortingunit import SplitSortingUnit
+import networkx as nx
```

Comment:
I think that the graph should be optional, so we don't enforce networkx installation. 

Note that `networkx` is installed in `full` installation, so most of the times it will be available, but if one only wants to use the core it should not be required.